php/values.yaml

```diff
@@ -305,9 +305,24 @@ fpm:
   # php.ini configuration
   # http://php.net/manual/en/errorfunc.configuration.php
   ini:
-    displayErrors: off
+    displayErrors: false
+    exposePhp: false
+    displayStartupErrors: false
+    htmlErrors: false
+    maxInputVars: 10000
+    mysqlnd.collectMemoryStatistics: true
+    mysqlnd.netReadTimeout: 10
+    requestOrder: GP
+    serializePrecisiOn: 17
+    sessiOn.gcMaxlifetime: 7200
+    shortOpenTag: false
+    trackErrors: false
+    variablesOrder: GPCS
+    zend.assertiOns: 0
     errorReporting: E_ALL
-    exposePhp: off
+    memoryLimit: 256M
+    postMaxSize: 200M
+    uploadMaxFilesize: 200M
```

Comment:
I think that it would be better to add only the necessary values here.

Reply:
Do you think the items added this time are unnecessary?